package/gluon-mesh-babel/files/etc/hotplug.d/iface/90-gluon-babel-brclient-no-igmp-snooping

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+case "$ACTION"
+in
+	ifup)
+	echo 1 > /sys/devices/virtual/net/br-client/bridge/multicast_snooping
```

Comment:
I only saw commit https://git.lede-project.org/?p=project/netifd.git;a=commitdiff;h=52541140f8138e31958cdc3d7e42a4029fa6bbc9, you should probably ask Felix about the reasons.